src/cozmo/opengl.py

```diff
@@ -24,7 +24,8 @@
     This package requires Python to have the PyOpenGL package installed, along
     with an implementation of GLUT (OpenGL Utility Toolkit).
 
-    To install the Python packages do ``pip3 install --user 'cozmo[3dviewer]'``
+    To install the Python packages do ``pip3 install --user cozmo[3dviewer]``
```

Comment:
I tried Windows, Bash and Zsh, and double quoutes seem to work in all of them, so I'd prefer it was just changed to:
pip3 install --user "cozmo[3dviewer]"